bluemira/equilibria/find.py

```diff
@@ -715,15 +715,21 @@ def get_flux_loop(psi_norm):
         f_s = find_flux_surf(x, z, psi, psi_norm, o_points=o_points, x_points=x_points)
         return Coordinates({"x": f_s[0], "z": f_s[1]})
 
-    low = 0.99  # Guaranteed (?) to be a closed flux surface
-    high = 1.01  # Guaranteed (?) to be an open flux surface
+    low = 1 - PSI_NORM_TOL  # Guaranteed (?) to be a closed flux surface
+    high = 1 + PSI_NORM_TOL  # Guaranteed (?) to be an open flux surface
```

Comment:
This could cause issues, in fact we start far away from one here to be pretty much certain we get a closed and open flux surface, when starting for the search of the LCFS.

Here, this isn't really the tolerance on psi_norm, but the offset from 1 from which to start the search of the LCFS. You could find that if you set psi_n_tol=1e-6, this procedure would just fail.

Reply:
In fact, this will definitely not work: `psi_n_tol` is already an input in this function and is used on the end tolerance below which we stop the procedure.